restify/restify.d.ts

```diff
@@ -452,6 +452,7 @@ declare module "restify" {
     responseTimeFormatter ?: (durationInMilliseconds: number) => any;
     handleUpgrades ?: boolean;
     router ?: Router;
+    httpsServerOptions?: Object;
```

Comment:
https://github.com/Microsoft/TypeScript-Handbook/blob/master/pages/declaration%20files/Do's%20and%20Don'ts.md#number-string-boolean-and-object
> If you're tempted to use the type Object, consider using any instead. There is currently no way in TypeScript to specify an object that is "not a primitive".